README.md

```diff
@@ -149,6 +149,20 @@ By the way, `HexInput` is also minimalist-friendly — only 400 bytes gzipped.
 
 </details>
 
+## TypeScript Support
+
+React-colorful supports TypeScript and ships with types in the library itself; no need for any other install.
+
+While not only typing its own functions and variables, it can also help you type yours. Depending on the component you're using, you can also import the type that is associated with the component. For example, if you're using the `HslColorPicker`, you can also import the `HSL` type.
```

Comment:
Since we don't have a component that "officially" called `HslColorPicker` I think it makes sense to update this text a bit. Maybe something like...
```diff
-if you're using the `HslColorPicker`, you can also import the `HSL` type.
+if you're using our HSL color picker component, you can also import the `HSL` type.
```